addons/sale_project/views/project_task_views.xml

```diff
@@ -100,7 +100,8 @@
         <field name="inherit_id" ref="project.view_task_search_form"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='partner_id']" position="after">
-                <field name="sale_order_id" string="Sale Order" filter_domain="['|', ('sale_order_id', 'ilike', self), ('sale_line_id', 'ilike', self)]"/>
+                <field name="sale_order_id" string="Sales Order" filter_domain="['|', ('sale_order_id', 'ilike', self), ('sale_line_id', 'ilike', self)]"/>
+                <field name="sale_line_id" string="Sales Order Item" filter_domain="[('sale_line_id', 'ilike', self)]"/>
```

Comment:
Those changes are no longer needed, can you remove them? 🙂 

Thanks in advance! 🙂